### ISSUE TITLE
ALL BLOCKS RARITY

### ISSUE THREAD

**Issue by KenzyHere7** [TiagoFar78/BlockBattles#36]
I don't know if it would be better to make a config for the rarities at each block and the priorities of the rarities, because it is possible that they will be modified

1. Custom_Model (Jack_o_lantern = Pumpkin) `Rare`
2. Chest `Common`
3. Oak door `Rare`
4. Oak Fence Gate `Common`
5. Oak Trapdoor `Rare`
6. Torch `Common`
7. Redstone Lamp `Common`
8. Flower Pot `Rare`
9. pointed dripstone `Common`
10. waxed exposed cut copper stairs `Common`
11. Custom_Model (Paper = Finder block) `Common`
12. daylight detector `Epic`
13. soul sand  `Epic`
14. Custom_Model (Paper = Boat)  `Epic`
15. Custom_model (Paper = Creeper Head) `Legendary`
16. iron trapdoor `Legendary`
17. monster spawner (without mob inside) `Common`
18. chain `Common`
19. block of quartz `Common`
20. white candle `Common`
21. iron_door `Rare`
22. iron bars `Common`
23. block of iron `Rare`
24. Custom_Model (Paper = Stick on the rock) `Rare`
25. Custom_Model (Paper = Higher) `Rare`
26. cobweb `Rare`
27. redstone_torch `Epic`
28. white bed `Epic`
29. Anvil `Legendary`

**Comment by TiagoFar78** [comment]
Why do you want rarities to? If its just to add it to the lore you can do it using the english.yml file. Is there any other thing you need rarity to?

**Comment by KenzyHere7** [comment]
> Why do you want rarities to? If its just to add it to the lore you can do it using the english.yml file. Is there any other thing you need rarity to?

For a future system crate, so we can do for example {random block common = 25% found} {random block legendary = 1%}

**Comment by TiagoFar78** [comment]
> For a future system crate, so we can do for example {random block common = 25% found} {random block legendary = 1%}

I already told you that crates system should be done using commands. It is not in the scope of this plugin. Any other thing?

**Comment by KenzyHere7** [comment]
> I already told you that crates system should be done using commands. It is not in the scope of this plugin. Any other thing?

yes i know, that was the reason, also for order in the gui, so people know what order to search in
